python/function.py

```diff
@@ -10,6 +10,11 @@ def __init__(
     ):
         # Tuples represent function types. (a, b, c) represents a -> b -> c.
         types = tuple([ty for _, ty in arguments] + [returntype])
+        if len(arguments) == 0:
+            types = ("unit",) + (returntype,)
```

Comment:
Isn't this just `types = ("unit", returntype)` or just `types = "unit", returntype`?